packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -1064,10 +1064,12 @@ export const getEntityHeaders = (
  * https://rest-docs.synapse.org/rest/GET/entity/alias/alias.html
  */
 export const getEntityAlias = (alias: string, accessToken?: string) => {
-  return doGet<EntityId>(
-    ENTITY_ALIAS(alias),
-    accessToken,
-    BackendDestinationEnum.REPO_ENDPOINT,
+  return allowNotFoundError(() =>
+    doGet<EntityId>(
+      ENTITY_ALIAS(alias),
+      accessToken,
+      BackendDestinationEnum.REPO_ENDPOINT,
+    ),
   )
 }
```

Comment:
This may return 404. Instead of throwing an error in that case, just return null